nbgitpuller/plugin_helper.py

```diff
@@ -184,8 +184,8 @@ async def handle_files_helper(helper_args, query_line_args):
     back to the origin
 
     :param dict helper_args: key-value pairs including the:
-        - download function
-        - download parameters in the case
+        - download_func download function
+        - download_func_params download parameters in the case
```

Comment:
I think params refer to the signature, while args refer to the actual things you pass to a function. Is this args?

Reply:
Yes nice catch. I am actually hating this whole download function abstraction mostly because it creates a very confusing set of moments about the temporary download directory. I am wondering if a better solution would be to mandate a directory be created by the plugin? This would get the creation of the directory out of plugin_helper and may limit the confusion for someone trying to create a custom downloader plugin.